src/central_freelist.cc

```diff
@@ -31,11 +31,11 @@
 // ---
 // Author: Sanjay Ghemawat <opensource@google.com>
 
-#include "config.h"
-#include <algorithm>
 #include "central_freelist.h"
+#include <algorithm>
+#include "config.h"
```

Comment:
config.h has to stay first. It is necessary (and was part of google c++ guide until some time ago when they started ignoring autotools).

Reply:
Oh, that was an automatic formatting reorder.
I will revert it.